tables/geography_groupings.md

```diff
@@ -1 +1,82 @@
-placeholder
+# tables/geography_groupings
+
+The `geography_groupings` table returns grouping categories for `geography_areas` which are identified via an `id`.
+
+The following `JOIN` queries can be carried out:
+
+- `geography_grouping_id` on the [geography_areas](geography_areas.md) table using `id`.
+
+## What are geography groupings?
+
+A `geography_groupings` refer to named collection/groupings of `geography_areas`, so if we look at the following query:
+
+```sql
+select ga.id, ga.description, ga.geography_code, ga.geography_grouping_id, gg.name as geography_grouping_name from geography_areas ga left join geography_groupings gg on gg.id = ga.geography_grouping_id where ga.id = 65;
+```
+
+We can see that `Down` has a geographic grouping of `Local Authorities`:
+
+|id|description|geography_code|geography_grouping_id|geography_grouping_name|
+|-|-|-|-|-|
+|65|Down|95NN|2,006|Local Authorities|
+
+## Example use
+
+You would like to retrieve the number of Local Authorities (`LA`) grouped by `top_level_geographies` along with the top-level `geography_code`:
+
+```sql
+select tlg.geography_code, tlg.description, count(ga.id) as local_authorities
+from geography_areas ga
+left join geography_groupings gg on ga.geography_grouping_id = gg.id
+left join top_level_geographies tlg on ga.top_level_geography_id = tlg.id
+where gg.abbreviation = 'LA' group by tlg.description, tlg.geography_code;
+
+```
+
+This results in the following:
+
+|geography_code|description|local_authorities|
+|--------------|-----------|-----------------|
+|E92000001|England|324|
+|N92000002|Northern Ireland|26|
+|S92000003|Scotland|32|
+|W92000004|Wales|22|
+
+
+## Schema
+
+|column|type|use|
+|-|-|-|
+|id|int4|Primary key.|
+|abbreviation|varchar(100)|Short code/abbreviation for a grouping, such as `CNTY` for `County`.|
+|name|varchar(100)|An end-user friendly name for a grouping, such as `Wards and Electoral Divisions`.|
+|description|text|An end-user friendly description for a grouping.|
+|top_level_geography_id|int4|The top-level geography for a given grouping, for example `Regions` has a top-level geography of `England` as the concept of a region exists in England only.|
```

Comment:
description of top_level_geography is good, I think that the `id` column in the schema needs as much description as this as they are of equal importance for the user to understand what this table is about

Reply:
Sure thing, will add more detail here.

Reply:
Detail added to `id`